pkgs/development/tools/misc/wishbone-tool/default.nix

```diff
@@ -1,19 +1,16 @@
 { lib, fetchFromGitHub, rustPlatform, libusb-compat-0_1 }:
 
-let
+rustPlatform.buildRustPackage rec {
+  pname = "wishbone-tool";
   version = "0.6.9";
-  src = fetchFromGitHub {
+  github = fetchFromGitHub {
```

Comment:
this should stay in the let in but I think we should keep the variable rename for clarity.

Reply:
i will revert the changes made to this file.